java/src/main/java/org/extism/sdk/manifest/MemoryOptions.java

```diff
@@ -1,11 +1,12 @@
 package org.extism.sdk.manifest;
 
-public record MemoryOptions(
+import com.google.gson.annotations.SerializedName;
 
-        /**
-         * Max number of pages.
-         */
-        Integer max // Integer to allow `null` as max
-//
-) {
+/**
+ * Configures memory for the Wasm runtime.
+ * Memory is described in units of pages (64KB) and represent contiguous chunks of addressable memory.
+ *
+ * @param maxPages Max number of pages.
+ */
+public record MemoryOptions(@SerializedName("max_pages") Integer maxPages) {
```

Comment:
@nilslice we decided to hold off on this change right? Just making sure.

Reply:
Yea, I think we will ship this as-is so it's immediately usable with the current release, and then we can make SDK-wide changes for the next release.

Reply:
by as-is, I mean using `"max"` as the name :)

Reply:
@thomasdarimont if you don't mind let's revert the change back to max for the time being.

Reply:
I renamed the `MemoryOptions` field back to `max`.